utils/emoticons.test.tsx

```diff
@@ -1,7 +1,9 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import * as Emoticons from 'utils/emoticons.jsx';
+/* eslint max-nested-callbacks: ["error", 10] */
```

Comment:
Hey @guigui64, I've made a fix for the `max-nested-callbacks` issue, which should now be in master. If you update your branch and do a fresh `npm install` it should pull in the new `eslintrc.json` with the fix. Then you can remove this exception.

Reply:
Thanks, I'm updating the PR right now ;)

Reply:
Actually, I still get the same issue after having synced...

`> mattermost-webapp@0.0.1 check /home/guillaume/git/mattermost-webapp
> eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx --ext tsx --ext ts . --quiet


/home/guillaume/git/mattermost-webapp/utils/emoticons.test.tsx
   31:76  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   32:85  error  Too many nested callbacks (4). Maximum allowed is 2  max-nested-callbacks
   38:54  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   43:70  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   48:67  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   53:60  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   58:54  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   63:70  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   68:70  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   73:76  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   78:76  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   83:87  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   88:87  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
   95:31  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
  100:30  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
  105:28  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
  106:37  error  Too many nested callbacks (4). Maximum allowed is 2  max-nested-callbacks
  111:43  error  Too many nested callbacks (4). Maximum allowed is 2  max-nested-callbacks
  117:30  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
  118:37  error  Too many nested callbacks (4). Maximum allowed is 2  max-nested-callbacks
  123:43  error  Too many nested callbacks (4). Maximum allowed is 2  max-nested-callbacks
  128:27  error  Too many nested callbacks (4). Maximum allowed is 2  max-nested-callbacks
  134:24  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks

/home/guillaume/git/mattermost-webapp/utils/url.test.tsx
  45:27  error  Too many nested callbacks (3). Maximum allowed is 2  max-nested-callbacks
  45:66  error  Too many nested callbacks (4). Maximum allowed is 2  max-nested-callbacks

✖ 25 problems (25 errors, 0 warnings)
`